src/libs/Navigation/getMatchingBottomTabRouteForState.ts

```diff
@@ -0,0 +1,28 @@
+// import CONST from '@src/CONST';
+import SCREENS from '@src/SCREENS';
+import getTopmostCentralPaneRoute from './getTopmostCentralPaneRoute';
+import TAB_TO_CENTRAL_PANE_MAPPING from './TAB_TO_CENTRAL_PANE_MAPPING';
+import type {BottomTabName, NavigationPartialRoute, RootStackParamList, State} from './types';
+
+// Get the route that matches the topmost central pane route in the navigation stack. e.g REPORT -> HOME
+function getMatchingBottomTabRouteForState(state: State<RootStackParamList>): NavigationPartialRoute<BottomTabName> {
+    const defaultRoute = {name: SCREENS.HOME};
+    const topmostCentralPaneRoute = getTopmostCentralPaneRoute(state);
+
+    if (topmostCentralPaneRoute === undefined) {
+        return defaultRoute;
+    }
+
+    for (const [tabName, centralPaneNames] of Object.entries(TAB_TO_CENTRAL_PANE_MAPPING)) {
+        if (centralPaneNames.includes(topmostCentralPaneRoute.name)) {
+            if (tabName === SCREENS.WORKSPACE.INITIAL) {
+                return {name: tabName, params: topmostCentralPaneRoute.params};
+            }
+            return {name: tabName as BottomTabName};
```

Comment:
It is not clear on why we only pass the params for the workspace initial page. Why we don't do that for all the pages?

Reply:
We have 3 tabs. Only this one needs policyID forwarded from the centralPaneScreens / RHPScreens.

We may need to revisit this place after merging with the workspace switcher.

Reply:
@adamgrzybowski do you still think we need to review this or should we keep the current code?

Reply:
We should keep it. WORKSPACE_INITIAL is the only screen in the bottom tab that needs to match the central pane.

e.g. we are opening a specific overview page and we need to open a matching workspace initial.

This is not happening for HOME and GLOBAL_SETTINGS.